examples/src/app/error.tsx

```diff
@@ -0,0 +1,41 @@
+'use client'
+
+function Samp(props: { children: React.ReactNode }) {
+  return <samp className="bg-red-100 p-1 rounded-md font-bold">{props.children}</samp>
+}
+
+export default function Error(props: { error: any }) {
+  let message = <p>Check the server console for more information.</p>
+
+  const errMsg = props.error.message
+  if (errMsg.startsWith('ServerRefused') && errMsg.includes('127.0.0.1')) {
```

Comment:
I don't like using string matching this way, especially when we have the data available as structured data inside the `props.error` value, unless I'm missing something?